server/controllers/temporary-accommodation/manage/bedspaceSearchController.test.ts

```diff
@@ -65,6 +65,7 @@ describe('BedspaceSearchController', () => {
         allPdus: referenceData.pdus,
         errors: {},
         errorSummary: [],
+        durationDays: '84',
```

Comment:
might be better to define a constant so that people will know and can be reused if needed and can maintain in once place

Reply:
Yup, I did create the constant and didn't reuse in test 🤦